examples/lock-app/nrfconnect/README.md

```diff
@@ -63,43 +111,29 @@ states are depicted:
     provisioned, but does not yet have full network (Thread) or service
     connectivity.
 
-*   _Solid On_ &mdash; The device is fully provisioned and has full network and
-    service connectivity.
-
-**Button #1** can be used to initiate a OTA software update as well as to reset
-the device to a default state.
-
-Pressing and holding Button #1 for 6 seconds initiates a factory reset. After an
-initial period of 3 seconds, all four LED will flash in unison to signal the
-pending reset. Holding the button past 6 seconds will cause the device to reset
-its persistent configuration and initiate a reboot. The reset action can be
-cancelled by releasing the button at any point before the 6 second limit.
+*   _Solid On_ &mdash; The device is fully provisioned and has full Thread
+    network and service connectivity.
 
 **LED #2** shows the state of the simulated lock bolt. When the LED is lit the
 bolt is extended (i.e. door locked); when not lit, the bolt is retracted (door
 unlocked). The LED will flash whenever the simulated bolt is in motion from one
 position to another.
 
-**Button #2** can be used to change the state of the simulated bolt. This can be
-used to mimick a user manually operating the lock. The button behaves as a
-toggle, swapping the state every time it is pressed.
+**Button #1** can be used to initiate a OTA software update as well as to reset
+the device to a default state.
 
-**Button #3** can be used to start Thread networking using default configuration
-which was selected to match OpenThread Border Router default settings and
-network credentials.
+**Button #2** can be used to manually change the state of the lock. Pushing the
+button is changing lock state to the opposite.
 
-**Button #4** can be used to start BLE advertisement, which is disabled by
-default.
+**Button #3** can be used to manually start Thread networking in the test mode,
+using default configuration.
 
-The remaining two LEDs (#3 and #4) are unused.
+**Button #4** can be used to start BLE advertisement.
```

Comment:
nit: I noticed that native speakers tend to use "BLE advertising" rather than "BLE advertisement" to name the activity which is kind of aligned with: https://wikidiff.com/advertising/advertisement. Maybe we should switch to the former?

Reply:
Ok, done.